vms/administrator/views.py

```diff
@@ -81,6 +77,7 @@ def post(self, request, *args, **kwargs):
 
 
 @login_required
+@admin_required
 def settings(request):
     user = request.user
     admin = None
```

Comment:
@necessary129  Do we need to check if the user is admin in the settings view now?